File: java/medidas.java
import java.util.Locale;
import java.util.Scanner;

public class medidas {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double a, b, c, quadrado, triangulo, trapezio;

	    System.out.print("Digite a medida A: ");
	    a = sc.nextDouble();

	    System.out.print("Digite a medida B: ");
	    b = sc.nextDouble();

	    System.out.print("Digite a medida C: ");
	    c = sc.nextDouble();


	    quadrado = a * a;
	    System.out.printf("AREA DO QUADRADO = %.4f\n", quadrado);

	    triangulo = (a * b) / 2;
	    System.out.printf("AREA DO TRIANGULO = %.4f\n", triangulo);

	    trapezio = ((a + b) * c) / 2;
	    System.out.printf("AREA DO TRAPEZIO = %.4f\n", trapezio);

		sc.close();
	}
}

File: python/medidas.py
a: float; b: float; c: float
quadrado: float; triangulo: float; trapezio: float

a = float(input("Digite a medida A: "))
b = float(input("Digite a medida B: "))
c = float(input("Digite a medida C: "))

quadrado = a * a
triangulo = (a * b) / 2
trapezio = ((a + b) * c) / 2

print(f"AREA DO QUADRADO = {quadrado:.4f}")
print(f"AREA DO TRIANGULO = {triangulo:.4f}")
print(f"AREA DO TRAPEZIO = {trapezio:.4f}")